tests/Microsoft.Identity.Test.Unit/WebUITests/WebView2WebUiFactoryTests.cs

```diff
@@ -29,6 +29,9 @@ public class WebView2WebUiFactoryTests
           new RequestContext(TestCommon.CreateDefaultAdfsServiceBundle(), Guid.NewGuid());
 
         [TestMethod]
```

Comment:
Consider adding a [TestCategory("Manual")] or smth like that.

Reply:
We can still run these on the classic pipeline so I think we can leave as is since these will still run automatically.